src/modules/dashboard/components/Dialogs/EditExpenditureDialog/ChangedBatch/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { Batch } from '~dashboard/ExpenditurePage/Batch/types';
+
+export const isBatchType = (obj: any): obj is Batch => {
+  return (
+    Object.prototype.hasOwnProperty.call(obj, 'recipients') ||
```

Comment:
Any reason for not using `obj.hasOwnProperty('recipients')` instead?

Reply:
When I do that, I have an error: 
![Screenshot 2022-10-14 at 10 08 10](https://user-images.githubusercontent.com/91876137/195796070-f8e6cd9b-8183-4f13-afba-2edbd3b5d1cc.png)

Reply:
Good to know, thanks